src/client/pfs.go

```diff
@@ -367,6 +367,21 @@ func (c APIClient) DeleteBranch(repoName string, branchName string, force bool)
 	return grpcutil.ScrubGRPC(err)
 }
 
+// ListRepoByType returns info about Repos of the given type
+// The if repoType is empty, all Repos will be included
+func (c APIClient) ListProject() (_ []*pfs.ProjectInfo, retErr error) {
+	ctx, cf := context.WithCancel(c.Ctx())
+	defer cf()
+	client, err := c.PfsAPIClient.ListProject(
+		ctx,
+		&pfs.ListProjectRequest{},
+	)
+	if err != nil {
+		return nil, grpcutil.ScrubGRPC(err)
```

Comment:
The scrub should probably be in a defer. The clientsdk function could return an unscrubbed error.